src/assets/db.json

```diff
@@ -353,5 +353,11 @@
     "logo": "https://static.lunar.fyi/svg/lunar.svg",
     "description": "Intelligent adaptive brightness for your external monitors.",
     "categories": ["Display", "Brightness", "XDR"]
+  },{
+    "id": 51,
+    "name": "RunCat",
```

Comment:
The application you suggested does not appear in the [brew --cask](https://formulae.brew.sh/cask/) API.